docs/docs/blocks/If.md

```diff
@@ -1,39 +0,0 @@
-# If
```

Comment:
How come this is gone?

Reply:
Looks like a mistake

Reply:
it's in a subfolder `!`, is that intentional?

Reply:
Yes. It is a hack for now.

If they stayed at the root, then all the folders would appear at the end in the navigation side-panel. I tried other characters (for instance `_` but that one is rendered as a white space so not helpful).

Try to move some files top the root and generate/serve the website, you will understand what I mean.

Reply:
Sounds good